packages/documentation/docs/controls/_dropdown_styleguide.md

```diff
@@ -16,32 +16,32 @@ For information about the element that opens the dropdown container, read the ar
 
 ## Options
 
-- **Header:** Add a header for the dropdown container. This provides a better understanding for the user what items are found in the dropdown.
+- **Header:** Add a header for the dropdown container. This helps the user to better understand which elements are in the dropdown.
 - **Quick Actions:** Add a quick action bar. We typically use quick actions to access common functions such as cut, copy and paste.
-- **Checked:** Mark selected items in the dropdown with a check mark.We typically use check marks if an item can be activated (e.g. for the dropdown "Sort by:" selection: Name, Change date, Create date).
+- **Checked:** Mark selected items in the dropdown with a check mark. We typically use check marks when an item can be activated (e.g. for the dropdown "Sort by:" selection: Name, Change date, Create date).
```

Comment:
Like that one! 2 small things:
1. Usually "Change date" is actually "Modified date"
2. I would keep this list of options lowercase, but that is maybe personal preference